lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -314,6 +315,22 @@ const { makePathsRelative, parseResource } = require("../util/identifier");
  * @property {ExtractorsByOption<Dependency, StatsModuleTraceDependency>} moduleTraceDependency
  */
 
+/**
+ * @param {string} readableIdentifier user readable identifier of the module
+ * @param {Module} module base module type
+ * @returns {string} an elided readableIdentifier, when readableIdentifier exceeds a maximum length
+ */
+const truncateLongExternalModuleReadableIdentifier = (
+	readableIdentifier,
+	module
+) => {
+	const maxLength = 80;
```

Comment:
Hoist this to the top of the file as a constant and use `CONSTANT_CASE` for the name. Reassigning continuously in this function adds unneeded memory pressure. 

I would probably go with something like: `MAX_EXTERNAL_MODULE_READABLE_IDENTIFIER_LENGTH`